README.md

```diff
@@ -1,158 +1,248 @@
+                    {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:GetBucketLocation",
+                        "s3:ListAllMyBuckets"
+                    ],
+                    "Resource": "arn:aws:s3:::*"
+                    },
+                    {
+                    "Effect": "Allow",
+                    "Action": ["s3:ListBucket"],
+                    "Resource" : { "Fn::Join" : [
+                        "", [ "arn:aws:s3:::", { "Ref" : "s3Bucket" }]]
+                        }
+                    },
+                    {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:PutObject",
+                        "s3:GetObject",
+                        "s3:DeleteObject"
+                    ],
+                    "Resource" : { "Fn::Join" : [
+                        "", [ "arn:aws:s3:::", { "Ref" : "s3Bucket" } , "/*" ]
+                    ] }
+                    }
+                ]
+                },
+            "PolicyName" : { "Fn::Join" : [
+                        "", [ { "Ref" : "s3Bucket" } , "ReadWriteAccess" ]
+                    ] }
+           }
+
+**s3ROAccessGroup**
+
+    Creates an IAM group with a policy that allows Read Only access to the Bucket.
+    Type: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-iam-group.html
+    Properties:
+        GroupName - S3bucketName-ReadOnlyAcess
+
+**s3BucketROAccessiamPolicy**
+     Groups: s3RWAccessGroup
+     PolicyDocument : {
+                    "Version": "2012-10-17",
+                    "Statement": [
+                        {
+                        "Effect": "Allow",
+                        "Action": [
+                            "s3:GetBucketLocation",
+                            "s3:ListAllMyBuckets"
+                        ],
+                        "Resource": "arn:aws:s3:::*"
+                        },
+                        {
+                        "Effect": "Allow",
+                        "Action": ["s3:ListBucket"],
+                        "Resource" : { "Fn::Join" : [
+                          "", [ "arn:aws:s3:::", { "Ref" : "s3Bucket" }]]
+							}
+						},
+						{
+                        "Effect": "Allow",
+                        "Action": [
+                            "s3:GetObject"
+                        ],
+                        "Resource" : { "Fn::Join" : [
+                          "", [ "arn:aws:s3:::", { "Ref" : "s3Bucket" } , "/*" ]
+                        ] }
+                        }
+                    ]
+                    },
+              "PolicyName" : { "Fn::Join" : [
+                          "", [ { "Ref" : "s3Bucket" } , "ReadOnlyAccess" ]
+                        ] }
+           }
+
+<a name="SpecialRequirements"></a>	
+#### Special Requirements
+
+
+<a name="Author"></a>	
+#### Original Author
+Jonathan Rudge (jonathan.rudge@kcom.com)
```

Comment:
Revert all change to this file

Reply:
Did not realise that got in! Will revert.